python/pyspark/sql/tests/test_utils.py

```diff
@@ -45,7 +45,14 @@
     IntegerType,
     BooleanType,
 )
-from pyspark.testing.sqlutils import have_pandas
+from pyspark.testing.sqlutils import (
+    have_pandas,
+    have_pyarrow,
+    pyarrow_requirement_message,
+)
+
+
+unittest.skipIf(not have_pyarrow, pyarrow_requirement_message)
```

Comment:
I think you should annotate the class below.

```python
@unittest.skipIf(...)
class UtilsTestsMixin
```

Reply:
Okay.